.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
 
       - name: Run linter
         run: composer run-script lint
+        if: matrix.php < '8.0'
```

Comment:
How come we don't want to run the linter on PHP8?

Reply:
php-cs-fixer does not run on PHP8 yet: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702